### DISCUSSION TITLE
Toggle some columns on click in q-table

### DISCUSSION THREAD

**Discussion by bagladivyang03** [quasarframework/quasar#9138] [General - Components / Directives / etc]
Hi all,
Is there any way in the quasar table to add a toggle(+) button such on the click it shows some hidden columns and then again on click hides i.e to its default state?
I tried adding the column in the table and then customising the header slot and added the event handling on click and hiding and showing the columns accordingly.

**Comment by metalsadman** [comment]
See docs examples https://quasar.dev/vue-components/table#visible-columns-custom-top-fullscreen
